.gitignore

```diff
@@ -67,3 +67,7 @@ cmake-build*
 # CBMC files
 *.goto
 *.log
+
+# Rust and Cargo
+*.lock
```

Comment:
There should only be a `Cargo.lock` at the root assuming you use workspaces (which it looks like you do).